lua/starfall/libs_sv/acf.lua

```diff
@@ -2227,6 +2245,51 @@ function ents_methods:acfDragCoef()
 	return ( this.BulletData[ "DragCoef" ] or 0 ) / ACF.DragDiv
 end
 
+--- Returns the heat of an ACF entity 
+-- @server
+-- @return number The heat value of the entity
+function ents_methods:acfHeat()
+	checktype(self, ents_metatable)
+	local this = unwrap(self)
+
+	if not (this and this:IsValid()) then
+		SF.Throw("Entity is not valid", 2)
+	end
+
+	local Heat
+	if isGun(this) then
+		Heat = ACE_HeatFromGun(this, this.Heat, this.DeltaTime)
+	elseif isEngine(this) then
+		Heat = ACE_HeatFromEngine(this)
+	else
+		Heat = ACE.AmbientTemp
+	end
+
+	return Heat
+end
+
+--- Returns all crewseats linked to an acf entity
+-- @server
+-- @return table crewseats entities
+function ents_methods:acfGetCrew()
+	checktype(self, ents_metatable)
+	local this = unwrap(self)
+
+	if not (this and this:IsValid()) then
+		SF.Throw("Entity is not valid", 2)
+	end
+
+	if restrictInfo( this ) then
+		return {}
+	end
+
+	local Crew = this.CrewLink
```

Comment:
This should be `local Crew = table.Copy(this.CrewLink)` or bad things will happen